sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/models/TableAccessPolicies.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables.models;
+
+import com.azure.core.annotation.Immutable;
+
+import java.util.List;
+
+/**
+ * This class contains values which correlate to the access polices set for a specific table.
+ */
+@Immutable
+public final class TableAccessPolicies {
+    private final List<TableSignedIdentifier> identifiers;
+
+    /**
+     * Constructs a {@link TableAccessPolicies}.
+     *
+     * @param identifiers {@link TableSignedIdentifier TableSignedIdentifiers} associated with the table.
+     */
+    public TableAccessPolicies(List<TableSignedIdentifier> identifiers) {
```

Comment:
Does Tables support public access as Blobs does?

Reply:
You mean like when you have a SAS for a given blob? Unfortunately I don't know that right now.